gradle/libs.versions.toml

```diff
@@ -28,6 +28,7 @@ androidxTracing = "1.3.0-alpha02"
 androidxUiAutomator = "2.2.0"
 androidxWindowManager = "1.2.0"
 androidxWork = "2.9.0"
+androidxUiTest = "1.7.0-alpha01"
```

Comment:
Also, `1.7.0-alpha03` is now the latest release - there were a couple of bug fixes for `DeviceConfigurationOverride` since `1.7.0-alpha01`, so it'd be nice to update to latest there.